docs/changes/0.27.0.rst

```diff
@@ -1,35 +1,67 @@
 Changelog for QCoDeS 0.27.0
 ===========================
 
-The June 2021 release of QCoDeS.
+The July 2021 release of QCoDeS.
 
 -----------------
 Breaking Changes:
 -----------------
 
-- DelegateInstrument takes `parameters` as an optional, keyworded input argument. Any arguments passed as positional arguments after this one will break with this change (#3089).
+- DelegateInstrument takes `parameters` as an optional, keyworded input argument. Any arguments passed as positional arguments after this one will break with this change. (#3089)
 
 ----
 New:
 ----
 
 - The default value for `default_parameter_name` in the Loop QCoDeS dataset changed from ``amplitude`` to ``None``.
   The ``amplitude`` value interferes with datasets where one of the setpoint arrays is called `amplitude`. (#3095)
+- Abstract parameters has been added which allow one to create abstract instrument types that are guaranteed 
+to have certain parameters present. (#3125)
+
+
 
 ---------
 Improved:
 ---------
 
 - DelegateInstrument can now have instrument channels. (#3089)
+- Decouple measurement name from dataset results table name to allow special characters in the measurement name. (#3185)
+- Update the get started document with installing and using spyder and jupyter. (#3179)
+- Add ``underlying_instrument`` property to ``_BaseParameter`` and use it in ``call_params_threaded`` instead of ``root_instrument``. (3127)
+- Ensure that the per array metadata is also exported to dataarrays and not just datasets. (#3116)
+- Use threaded setting from config in doNd functions, and make ``process_params_meas`` public. (#3096)
+
 
 -----------------
 Improved drivers:
 -----------------
 
--
+- Fixed bug in RTO1000 digitial oscilloscope by forwarding instrument to parameter baseclass. (#3205)
```

Comment:
This has no user facing changes and is equivalent to the changes done in other drivers via #3187